modules/navigation/src/main/res/layout-land/layout_navigation.xml

```diff
@@ -49,16 +49,28 @@
         android:focusable="true">
 
         <ViewStub
-            android:id="@+id/signpostViewViewStub"
-            android:inflatedId="@+id/signpostView"
-            android:layout_width="344dp"
+            android:id="@+id/signpostView"
+            android:layout_width="@dimen/signpostViewWidth"
             android:layout_height="wrap_content"
+            android:inflatedId="@id/signpostView"
             android:layout="@{navigationFragmentViewModel.signpostLayout}"
             android:visibility="@{safeUnbox(navigationFragmentViewModel.signpostEnabled) ? View.VISIBLE : View.GONE}"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
             app:navigationFragmentViewModel="@{navigationFragmentViewModel}" />
 
+        <ViewStub
+            android:id="@+id/lanesView"
+            android:layout_width="@dimen/lanesViewWidth"
+            android:layout_height="wrap_content"
+            android:inflatedId="@id/lanesView"
+            android:layout="@layout/layout_lanes_view_stub"
+            android:visibility="@{safeUnbox(navigationFragmentViewModel.lanesViewEnabled) ? View.VISIBLE : View.GONE}"
```

Comment:
tu by měl být ještě i zároveň check na ten SignpostType.FULL a signpostEnabled hodnotu, až to budeš mít jeho součástí.